spring-jms/src/test/com/baeldung/spring/jms/DefaultTextMessageSenderTest.java

```diff
@@ -0,0 +1,26 @@
+package com.baeldung.spring.jms;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import com.baeldung.spring.jms.SampleJmsMessageSender;
+
+public class DefaultTextMessageSenderTest {
+
+    private SampleJmsMessageSender messageProducer;
+
+    @SuppressWarnings("resource")
+    @Before

```

Comment:
"@Before" runs before every method. "@BeforeClass" runs once before running all tests. I guess we need to second option in this case